app/connectors/ProxiedHttpClient.scala

```diff
@@ -37,13 +36,11 @@ class ProxiedHttpClient @Inject()(config: Configuration,
                                   runMode: RunMode)
   extends DefaultHttpClient(config, httpAuditing, wsClient, actorSystem) with WSProxy {
 
-  val authorization: Option[Authorization] = None
   val apiKeyHeader: Option[(String, String)] = None
   private val env = runMode.env
 
-  def withHeaders(bearerToken: String, apiKey: String = ""): ProxiedHttpClient = {
+  def withHeaders(apiKey: String = ""): ProxiedHttpClient = {
     new ProxiedHttpClient(config, httpAuditing, wsClient, environment, actorSystem, runMode) {
-      override val authorization = Some(Authorization(s"Bearer $bearerToken"))
```

Comment:
This seems to have been left from when we moved from mergeroo to ebridge. Before, we used the authorization header, whereas with ebridge we use the api-key to go across environments. The authorization header is now also used for calls between microservices in the same environment (for sensitive enpoints), so this gets in the way.